content/projects/_index.md

```diff
@@ -25,15 +25,11 @@ how close they are to the [UNAIDS 90-90-90 target](https://www.unaids.org/en/res
 
 * [Paper describing the model](https://www.biorxiv.org/content/10.1101/532010v1)
 
-### Naomi
+### [Naomi](/projects/naomi)
```

Comment:
This is too subtle in the page.  Something like `see the [project page][/projects/naomi]` for more information, though `[naomi]` would probably work for the link